src/auto_code_demogs.py

```diff
@@ -50,6 +50,36 @@ def auto_code_demogs(cls, user, data, phone_uuid_table, coda_output_dir):
                 )
             td.append_data({"operator_coded": label.to_dict()}, Metadata(user, Metadata.get_call_location(), time.time()))
 
+        # Subsample messages for export to coda
+        subsample_data = MessageFilters.subsample_messages(data)
+
+        # Output single-scheme subsample answers to coda for manual verification + coding
+        IOUtils.ensure_dirs_exist(coda_output_dir)
+        for plan in PipelineConfiguration.DEMOG_CODING_PLANS:
+            if plan.raw_field == "location_raw":
+                continue
+            
+            TracedDataCodaV2IO.compute_message_ids(user, subsample_data, plan.raw_field, plan.id_field)
+
+            coda_output_path = path.join(coda_output_dir, f'sub_sample_{plan.coda_filename}')
+            with open(coda_output_path, "w") as f:
+                TracedDataCodaV2IO.export_traced_data_iterable_to_coda_2(
+                    subsample_data, plan.raw_field, plan.time_field, plan.id_field, {plan.coded_field: plan.code_scheme}, f
+                )
+        
+        # Output subsample location scheme to coda for manual verification + coding
+        output_path = path.join(coda_output_dir, "sub_sample_location.json")
```

Comment:
For now we need to be exporting the whole of the demogs, and only subsampling the messages

Reply:
Noted thanks!